src/test/java/com/programmers/application/domain/FixedAmountVoucherTest.java

```diff
@@ -0,0 +1,47 @@
+package com.programmers.application.domain;
+
+import com.programmers.application.domain.voucher.Voucher;
+import com.programmers.application.domain.voucher.VoucherFactory;
+import com.programmers.application.dto.request.RequestFactory;
+import com.programmers.application.dto.request.VoucherCreationRequest;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class FixedAmountVoucherTest {
+    private static final int MIN_DISCOUNT_AMOUNT = 0;
+    private static final int MAX_DISCOUNT_AMOUNT = 5000;
+    private static final String VOUCHER_TYPE = "fixed";
+
+    @DisplayName("고정 할인 바우처 discount() 메서드 성공 테스트")
+    @ParameterizedTest
+    @CsvSource(value = {"100, 1000, 900", "500, 1000, 500"})
+    void discount(long discountAmount, long originalPrice, long expectedPrice) {
+        VoucherCreationRequest voucherCreationRequest = RequestFactory.createVoucherCreationRequest(VOUCHER_TYPE, discountAmount);
+        Voucher voucher = VoucherFactory.createVoucher(voucherCreationRequest);
+        assertThat(voucher.discount(originalPrice)).isEqualTo(expectedPrice);
```

Comment:
[nit] 
```suggestion
        VoucherCreationRequest voucherCreationRequest = RequestFactory.createVoucherCreationRequest(VOUCHER_TYPE, discountAmount);
        Voucher voucher = VoucherFactory.createVoucher(voucherCreationRequest);
        
        int discountPrice = voucher.discount(originalPrice);
        
        assertThat(discountPrice).isEqualTo(expectedPrice);
```
요게 좀 더 가독성 좋은 것 같아요!